onnxruntime/test/onnx/main.cc

```diff
@@ -29,7 +29,7 @@ using namespace onnxruntime;
 
 namespace {
 void usage() {
-  auto version_string = ToUTF8String(OrtGetApiBase()->GetVersionString());
+  auto version_string = Ort::GetVersionString();
```

Comment:
For example, here you pass a std::string directly to printf, it works only if the std::string only contains ASCII chars. But most people are not aware of that.

Reply:
We define all strings as UTF-8 internally, except Windows paths. Version string will always be ASCII.